### PR TITLE
EPMRPP-78391 || create new Plugins section to add SAML Plugin doc

### PR DESCRIPTION
**Pull request by pressayuliya**
create new Plugins section to add SAML Plugin doc and move other plugins

### CODE REVIEW

**Review by Vadim73i — APPROVED** [review]

File: src/md/src/Plugins/SAMLPlugin.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+***ReportPortal SSO initial URL***
+
+You need to provide a URL for a SAML Provider to deliver SAML data for the identity federation.
+
+```url
+https://<host>/uat/saml/sp/SSO/alias/report-portal-sp
+```
+#### ReportPortal configuration
+
+[ ![ImageName](Images/Plugins/SAMLPlugin/SAML-plugin.png) ](Images/Plugins/SAMLPlugin/SAML-plugin.png)
```
Comment (raikbitters) [inline src/md/src/Plugins/SAMLPlugin.md:44]: an image isn't exist

File: src/md/src/Plugins/SAMLPlugin.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+***Identity provider name ID (Optional)***
+
+**Identity provider name ID** (aka name identifier formats) controls how the users at identity providers are mapped to users at service providers.
+
+We support next formats:
+
+```
```
Comment (raikbitters) [inline src/md/src/Plugins/SAMLPlugin.md:52]: Proposal:

UNSPECIFIED - used by default

```URL
urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified
```

EMAIL

```URL
urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress
```

PERSISTENT

```URL
urn:oasis:names:tc:SAML:2.0:nameid-format:persistent 
```

File: src/md/src/Plugins/SAMLPlugin.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Attribute name from SAML metadata which contains an user last/family name.
+
+```
+<saml:Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
+<saml:AttributeValue xsi:type="xs:string">Gibson</saml:AttributeValue>
+</saml:Attribute>
+```
+
+***Full name***
```
Comment (raikbitters) [inline src/md/src/Plugins/SAMLPlugin.md:114]: Add a new paragraph

File: src/md/src/Plugins/SAMLPlugin.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+```
+https://<host>/uat
+```
+
+***First name***
+
+Attribute name from SAML metadata which contains an user first/given name.
+
+```
```
Comment (raikbitters) [inline src/md/src/Plugins/SAMLPlugin.md:98]: add xml marker

like this ```xml

File: src/md/src/Plugins/SAMLPlugin.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+<saml:Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
+<saml:AttributeValue xsi:type="xs:string">William</saml:AttributeValue>
+</saml:Attribute>
+```
+
+***Last name***
+
+Attribute name from SAML metadata which contains an user last/family name.
+
+```
```
Comment (raikbitters) [inline src/md/src/Plugins/SAMLPlugin.md:108]: add xml marker

like this ```xml

File: src/md/src/Plugins/SAMLPlugin.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+<saml:Attribute Name="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri">
+<saml:AttributeValue xsi:type="xs:string">Gibson</saml:AttributeValue>
+</saml:Attribute>
+```
+
+***Full name***
+Attribute name from SAML metadata which contains a full user name. You can use either two separate attributes for first name and last name or a combined first and last name attribute. This solely depends on your SAML provider.
+
+```
```
Comment (raikbitters) [inline src/md/src/Plugins/SAMLPlugin.md:117]: add xml marker

like this ```xml

File: src/md/src/Plugins/SAMLPlugin.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+ </saml:Attribute> 
+```
+
+***ReportPortal Callback URL (Optional)***
+
+This field provides a redirect base path.
+
+Once you have submitted an integration with “RP callback URL”, the URL will be applied to all SAML integrations.
+
+```
```
Comment (raikbitters) [inline src/md/src/Plugins/SAMLPlugin.md:90]: add url marker

like this  ```url

File: src/md/src/Plugins/SAMLPlugin.md (hunk 3 above)
Comment (raikbitters) [reply src/md/src/Plugins/SAMLPlugin.md:114]: This looks like:
***Full name*** Attribute name from SAML metadata

**Pull request merged**